firmware/controllers/sensors/redundant_sensor.cpp

```diff
@@ -9,33 +9,66 @@ RedundantSensor::RedundantSensor(SensorType outputType, SensorType first, Sensor
 {
 }
 
-void RedundantSensor::configure(float maxDifference, bool ignoreSecondSensor) {
+void RedundantSensor::configure(float maxDifference, bool ignoreSecondSensor, float secondMaximum) {
 	m_maxDifference = maxDifference;
 	m_ignoreSecond = ignoreSecondSensor;
+	m_secondMaximum = secondMaximum;
 }
 
 SensorResult RedundantSensor::get() const {
-	auto result1 = Sensor::get(m_first);
+	// Sensor 1 is always full range, i.e. 0% -> 100%
+	auto sensor1 = Sensor::get(m_first);
+
+	// If sensor 1 result is invalid, return inconsistency error
+	if (!sensor1.Valid) {
```

Comment:
I don't think it's correct to put this above the `m_ignoreSecond` check - in single sensor mode, we want to return the true error instead of `Inconsistent`

Reply:
I updated the unit tests to expect this; they should fail here momentarily and subsequent commit will fix this behavior.  Good catch; thank you.

Reply:
https://github.com/FOME-Tech/fome-fw/actions/runs/9354363787/job/25747069591?pr=430#step:8:25135
```
[==========] 595 tests from 120 test suites ran. (387 ms total)
[  PASSED  ] 592 tests.
[  FAILED  ] 3 tests, listed below:
[  FAILED  ] SensorRedundantIgnoreSecond.OnlyFirst
[  FAILED  ] SensorRedundantIgnoreSecond.OnlySecond
[  FAILED  ] SensorRedundantIgnoreSecond.SetBothIgnoreSecond
```